src/test/java/de/blackforestsolutions/dravelopsdatamodel/objectmothers/ApiTokenObjectMother.java

```diff
@@ -315,13 +321,29 @@ public static ApiToken getRnvGtfsApiToken() {
                 .build();
     }
 
-    public static ApiToken getConfiguredStationPersistenceApiToken() {
+    public static ApiToken getConfiguredTravelPointPersistenceApiToken() {
+        return getConfiguredStationPersistenceApiToken()
+                .setPath("/travelpoints/get")
+                .build();
+    }
+
+    public static ApiToken getConfiguredPolygonPersistenceApiToken() {
+        return getConfiguredStationPersistenceApiToken()
+                .setPath("/geocoding/get/operatingPolygon")
+                .build();
+    }
+
+    public static ApiToken getConfiguredBoxPersistenceApiToken() {
+        return getConfiguredStationPersistenceApiToken()
+                .setPath("/geocoding/get/operatingBox")
+                .build();
+    }
+
+    private static ApiToken.ApiTokenBuilder getConfiguredStationPersistenceApiToken() {
         return new ApiToken.ApiTokenBuilder()
                 .setProtocol("http")
                 .setHost("localhost")
-                .setPort(8086)
-                .setPath("/travelpoints/get")
-                .build();
+                .setPort(8086);
```

Comment:
Habe ich selber aus relativ harter erfahrung gelernt, Magic Strings und Numbers in einer Objectmother erhöhen den wartungsaufwand sehr stark (wenn es nicht sicher einmalige werte sind.

Reply:
Heisst das, dass die Klasse voll mit Konstanten sein soll?

Reply:
Ich habe die ganze Klasse mal voll mit Konstanten gemacht

Reply:
Done